scripts/change_url

```diff
@@ -108,7 +109,7 @@ fi
 # UPDATE CONFIGURATION FILE
 #=================================================
 
-ynh_exec_warn_less 'ynh_add_config --template="../conf/config.xml" --destination="$final_path/.data/config.xml"'
```

Comment:
The `ynh_exec_warn_less` was there to avoid the warning about the file being manually modified. Do you have an idea how to avoid that warning altogether?